### ISSUE TITLE
Mouse buttons stop working under certain circumstances. 

### ISSUE THREAD

**Issue by Bootrot** [richardwilkes/gcs#369]
If I manually change any value on a character sheet (type in new data), and click on any randomizer buttons (name, age, hair etc.) while the manually changed field still has focus, the mouse buttons stop working (on anything but the close window X). If I change a value and click the focus off of that field, and then click a randomizer, there is no issue. Or It's just me. Windows 10, if that's helpful. Thanks... or sorry if it IS just me.

**Comment by richardwilkes** [comment]
Thanks. This might help me track down that particular problem. I've been unable to reliably reproduce it in the past, which makes it really difficult to fix.

**Comment by Awesome456** [comment]
I've experienced the same issue. I noticed though that the Save changes prompt still appears if you attempt to close out of the program, so the program doesn't do a complete freeze thanks to this bug. The freeze seems to be very particular. An example I've gotten while testing was as follows:

I make a new character sheet and begin with randomizing the name. Afterwards I focus the field but make no changes to it, merely putting my cursor at the end of the prompt. From here I can randomize a different field with no difficulties. 

However the moment I focus a field and make a change (deleting or inserting new characters), I am unable to use any of the randomizer buttons without freezing the program up. 

Removing a character and then inserting the same character again seems to not cause any problems interestingly enough.

The freeze seems to be related to whatever was previously "remembered" in the currently focused field but that's just a guess.
